docs/source/guides/worklock_guide.rst

```diff
@@ -52,12 +324,153 @@ You can obtain information about the current state of WorkLock by running:
     (nucypher)$ nucypher worklock status --network <NETWORK> --provider <YOUR PROVIDER URI>
 
 
-If you want to see detailed information about your current bid, you can specify your bidder address with the ``--bidder-address`` flag:
+The following is an example output of the ``status`` command (hypothetical values):
+
+.. code::
+
+     _    _               _     _                   _
+    | |  | |             | |   | |                 | |
+    | |  | |  ___   _ __ | | __| |      ___    ___ | | __
+    | |/\| | / _ \ | '__|| |/ /| |     / _ \  / __|| |/ /
+    \  /\  /| (_) || |   |   < | |____| (_) || (__ |   <
+     \/  \/  \___/ |_|   |_|\_\\_____/ \___/  \___||_|\_\
+
+    ══ <NETWORK> ══
+
+    Reading Latest Chaindata...
+
+    Time
+    ══════════════════════════════════════════════════════
+
+    Contribution (Closed)
+    ------------------------------------------------------
+    Claims Available ...... Yes
+    Start Date ............ 2020-03-25 00:00:00+00:00
+    End Date .............. 2020-03-31 23:59:59+00:00
+    Duration .............. 6 days, 23:59:59
+    Time Remaining ........ Closed
+
+    Cancellation (Open)
+    ------------------------------------------------------
+    End Date .............. 2020-04-01 23:59:59+00:00
+    Duration .............. 7 days, 23:59:59
+    Time Remaining ........ 1 day, 2:47:32
+
+
+    Economics
+    ══════════════════════════════════════════════════════
+
+    Participation
+    ------------------------------------------------------
+    Lot Size .............. 280000000 NU
+    Min. Allowed Bid ...... 15 ETH
+    Participants .......... 1000
+    ETH Supply ............ 50000 ETH
+    ETH Pool .............. 50000 ETH
+
+    Base (minimum bid)
+    ------------------------------------------------------
+    Base Deposit Rate ..... 1000 NU per base ETH
+
+    Bonus (surplus over minimum bid)
+    ------------------------------------------------------
+    Bonus ETH Supply ...... 35000 ETH
+    Bonus Lot Size ........ 265000000 NU
+    Bonus Deposit Rate .... 7571.43 NU per bonus ETH
+
+    Refunds
+    ------------------------------------------------------
+    Refund Rate Multiple .. 4.00
+    Bonus Refund Rate ..... 1892.86 units of work to unlock 1 bonus ETH
+    Base Refund Rate ...... 250.0 units of work to unlock 1 base ETH
+
+        * NOTE: bonus ETH is refunded before base ETH
+
+
+For the less obvious values in the output, here are some definitions:
+
+    - Lot Size
+        NU tokens to be distributed by WorkLock
+    - ETH Supply
+        Sum of all ETH bids that have been placed
+    - ETH Pool
+        Current ETH balance of WorkLock that accounts for refunded ETH for whales i.e. `ETH Supply` - `Whale Refunds`
```

Comment:
Whale refund will change ETH supply, ETH Pool is ETH Supply - regular refund (after completing work)

Reply:
✔️